monitoring/uss_qualifier/configurations/dev/library/environment.yaml

```
@@ -26,6 +27,7 @@ net_rid:
             service_providers:
                 - participant_id: uss1
                   injection_base_url: http://host.docker.internal:8081/ridsp/injection
+                  uss_base_urls: [ http://host.docker.internal:8081 ]
```

For the sake of consistency, the base URL should be defined as the whole URL up to path defined in the OpenAPI definition:
```suggestion
                  uss_base_urls: [ http://host.docker.internal:8081/mock/ridsp/v2 ]
```
(in the case of the mock USS, this suffix is hard-coded)